File: hello.rb
#!/usr/bin/env ruby

def env(name)
  ENV[name] || ''
end

puts "Content-Type: text/plain"
puts
puts "Hello from CGI 👋"
puts "REQUEST_METHOD = #{env('REQUEST_METHOD')}"
puts "QUERY_STRING   = #{env('QUERY_STRING')}"
puts "SCRIPT_NAME    = #{env('SCRIPT_NAME')}"
puts "CONTENT_TYPE   = #{env('CONTENT_TYPE')}"
puts "CONTENT_LENGTH = #{env('CONTENT_LENGTH')}"

File: hello.py
#!/usr/bin/env python3
import os
import html

print("Content-Type: text/plain")
print()
print("Hello from CGI 👋")
print(f"REQUEST_METHOD = {os.environ.get('REQUEST_METHOD','')}")
print(f"QUERY_STRING   = {os.environ.get('QUERY_STRING','')}")
print(f"SCRIPT_NAME    = {os.environ.get('SCRIPT_NAME','')}")
print(f"CONTENT_TYPE   = {os.environ.get('CONTENT_TYPE','')}")
print(f"CONTENT_LENGTH = {os.environ.get('CONTENT_LENGTH','')}")

